include/Machine.h

```diff
@@ -42,7 +46,7 @@ class Machine
   bool login(FabMember user); // if the machine is not active, login the user
   void logout();              // if the machine is active, check if the card belongs to the user that is logged in and logout the user
   unsigned long getUsageTime();
-  MachineIDType getMachineId();
+  uint16_t getMachineId();
```

Comment:
the use of a specific type is better; let the compiler help you to not mix number and ID.
But you will probably have to have a struct rather than a typedef for the typesystem to work properly. As the struct will contain only one field, it should be compiled away anyway, so it come for free